### ISSUE TITLE
Remove recommended URI patterns

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1214]
As they're not very valuable and take up time to understand and subsequently ignore.

**Comment by jpstroop** [comment]
+💯. I don't really see why we need them at all, add they'll only be a nuisance if we ever get around to specing out IIIF CRUD/LDP.

**Comment by zimeon** [comment]
:+1: agreed, I've felt rather awkward about them for some time as they are appear overly prescriptive (until one realizes they can be ignored)

**Comment by mikeapp** [comment]
👍 I feel like I've seen discussions on Slack or iiif-discuss where people have been confused about whether the patterns are mandatory

**Comment by azaroth42** [comment]
Re-propose, following @jpstroop, that we don't need them at all and they can be dropped completely?

**Comment by tomcrane** [comment]
👍 to drop completely.

**Comment by glenrobson** [comment]
One thing I would raise is that its difficult to make a generic service that offers services related to a manifest as its difficult to mint a useful id. For example in SAS I wanted to offer a IIIF Search Service for the following manifest:

http://dams.llgc.org.uk/iiif/2.0/4642022/manifest.json

As the manifest implements the recommended URI pattern:

```{scheme}://{host}/{prefix}/{identifier}/manifest```

I was able to walk back through the URI (splitting on ```/```) and get the identifier 4642022 and offer the following:

http://dams.llgc.org.uk/iiif/4642022/search

I know this won't work on all cases but it was useful to create a search service that related to the manifest id. The alternative was just to mint a UID for the search service but this would have made the manifest generation more difficult as it would need a lookup to convert the UID back to a manifest ID.

A more sustainable solution would be if the manifest advertised a meaning ID (maybe a slug) that should be used by consuming services for identification. Something that works in a URI as the full manifest ```@id``` doesn't work unless its being passed as a parameter to a service which goes against the resource pattern. 

If the recommend URI pattern does go it would be useful to have an alternative way of identifying manifests with a short id. @tomcrane, have you come across this with Elucidate or your IIIF Search software?

**Comment by zimeon** [comment]
:+1: to dropping completely.

To @glenrobson's point: I think we need to be clear on how clients can follow-their-noses (or the linked-data to be more specific) to discover related resources and services. For example using the [search service description](http://iiif.io/api/search/1.0/#service-description) to find the search service.

**Comment by zimeon** [comment]
:+1: agreed at Toronto WG meeting

**Comment by ahankinson** [comment]
I flagged a few issues at the meeting that I would like to restate here. As an implementer I found the recommended URI patterns useful, not particularly because of their application to the IIIF API, but in their ability to communicate URI design best-practices in general. It would be a real detriment to the community to start seeing URLs in the place of URIs, and for institutions to start publishing IDs like:

`http://example.com/canvas?id=12345`

Or, even worse:

`http://example.com/imageserver.fcgi?IIIF=/{identifier}/{region}/{size}/{rot}/{quality}.{ext}`

If they are removed, I think they should probably be replaced with a discussion on URI/URL construction best practices, including things like ensuring your URLs don't change once they are published.
